include/boost/geometry/strategies/geographic/buffer_side_straight.hpp

```diff
@@ -52,21 +54,23 @@ public :
     template
     <
         typename Point,
-        typename OutputRange,
+        typename RangeOut,
```

Comment:
Is there a reason why template arguments are in different order than function arguments?

Reply:
Thanks, no, I'll update it.

Reply:
:heavy_check_mark: